Automattic/wp-calypso
# Update @wordpress/edit-site

#### Proposed Changes

This PR updates `@automattic/design-preview` dependency of `@wordpress/edit-site` from `4.6.0` to `4.17.0`, which contains the fixes:
- https://github.com/WordPress/gutenberg/pull/43601
- https://github.com/WordPress/gutenberg/pull/44556

Which, in turn, fixes this UI issue for the `<DesignPreview />` component:
| Before | After |
| --- | --- |
| ![Screen Shot 2022-10-19 at 5 44 11 PM](https://user-images.githubusercontent.com/797888/196657111-41f92fe4-9053-4faf-957b-d9634ce1380c.png) | ![Screen Shot 2022-10-19 at 5 44 38 PM](https://user-images.githubusercontent.com/797888/196657125-ddca33d8-1989-4779-894a-529b57d1eb2f.png) |


#### Testing Instructions



* Head to the design picker `setup/designSetup?siteSlug=${site_slug}`
* Preview any design with style variations.
* Ensure that the style preview doesn't have extra padding (happened previously for the theme "Pixl"). 

#### Pre-merge Checklist
